compose.yaml

```diff
@@ -6,7 +6,7 @@ configs:
 
 services:
```

Comment:
Please remove the Docker compose file, we prefer not to set up things manually anymore and use Nomad or Kubernetes. For development, it's fine to use Docker compose, but the compose file was removed because we would not like to use it anymore.